pkg/pb/irreparabledb.proto

```diff
@@ -0,0 +1,69 @@
+// Copyright (C) 2018 Storj Labs, Inc.
+// See LICENSE for copying information.
+
+syntax = "proto3";
+option go_package = "pb";
+
+package irreparabledb;
+
+// IrrSegDB Irreparable segment DB defines the interface for retrieving and updating irreparable segments 
+service IrrSegDB {
```

Comment:
All of these should use full names, not shorthands. So `IrreparableSegmentDB`, `RemoteSegmentInfo`, `PutIrreparableSegmentRequest` etc.

Reply:
Removed the protocol buffer files as suggested